modules/UpgradeWizard/uw_ajax.php

```diff
@@ -892,8 +892,8 @@ function systemCheckJsonCheckFiles($persistence)
                 // don't warn yet - we're going to use this to check against replacement files
                 $filesNotWritable[$i] = $file;
                 $filesNWPerms[$i] = substr(sprintf('%o', fileperms($file)), -4);
-                $owner = posix_getpwuid(fileowner($file));
-                $group = posix_getgrgid(filegroup($file));
+                $owner = (function_exists('posix_getpwuid') ? posix_getpwuid(fileowner($file)) : array('name' => 'NULL'));
```

Comment:
is `NULL` string enough information to figure out what happends here? Please add a log notes to describe the problem at least.